optuna/visualization/_pareto_front.py

```diff
@@ -138,15 +138,33 @@ def _get_pareto_front_2d(
             text=[_make_hovertext(t) for t in trials[len(study.best_trials) :]],
             mode="markers",
             hovertemplate="%{text}<extra>Trial</extra>",
-            name="Trial",
+            marker={
+                "line": {"width": 0.5, "color": "Grey"},
+                "color": [t.number for t in trials[len(study.best_trials) :]],
```

Comment:
IIUC, `study.best_trials` could be a somewhat heavy call when there are too many trials, so how about storing the best trials into a variable at the head of this function and reusing that here (and other places)?

Reply:
Good catch, thanks. It should be fixed in the latest commit.

Reply:
Thanks!